db/seeds.d/010-calibrate_bcrypt_cost.rb

```diff
@@ -0,0 +1,11 @@
+# calibrate default cost to be at least 5ms (performance-wise safe)
+new_cost = BCrypt::Engine.calibrate(5)
```

Comment:
Perhaps the user should be able to override the 5 here? I've always been taught to at least use constants for magic values or make them configurable.

Reply:
I lean towards less complex interface for users. Now there is just "cost" value exposed, you want to expose "cost" and also "calibration time" which actually sets "cost" and it's hard to explain. I'd rather keep this as constant, user can always increase cost and the seed will not touch it anymore.